src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,72 @@
+package duke.ui;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.Scanner;
+
+/**
+ * Represents the user interface of the application.
+ */
+public class Ui {
+    private static final String SEP = "\t__________________________________________";
+    private final Scanner in;
+    private final PrintStream out;
```

Comment:
Constant names should be all uppercase using underscore to separate words
```suggestion
    private final Scanner IN;
    private final PrintStream OUT;
```